### ISSUE TITLE
[Feature Request] chrome watcher

### ISSUE THREAD

**Issue by amitamitamitamit** [ActivityWatch/aw-android#74]
Is there a chrome watcher like in the desktop version?

**Comment by ErikBjare** [comment]
Merged but unreleased: https://github.com/ActivityWatch/aw-android/pull/30

aw-android development has stalled a bit, but hope to throw together a new version in the next couple months.

**Comment by amitamitamitamit** [comment]
Is there a ready to use apk that I can download? I have small experience with android development but not so much.
Thanks for your helping

**Comment by ErikBjare** [comment]
Nope, not at the moment. Currently working on giving aw-android a proper update in the coming month.

**Comment by amitamitamitamit** [comment]
Ok, thank you.
I saw you wrote that you will check if you can create also a firefox watcher, it will be with the release?
